MovingBlocks/Terasology
# chore(world): remove unused AbstractChunkView.java

### Contains

Remove the unused abstract class `AbstractChunkView.java`. It is not used in the complete Omega workspace, so I assume it is safe to remove.

### How to test

Checkout this engine PR in an Omega workspace and verify that everything still compiles.

---

Cherry-picked from bd9b163 (#4384)
